modules/federal/assets/configs/systemsConfig.json

```diff
@@ -1,14 +1,16 @@
 {
     "hard": {
     	"hard": true,
-        "constantAllies": [
+        "constantAllies":
+        [
             {
                 "hull": "federal:federalCapital",
                 "items": "federal:dualBlaster federal:dualBlaster core:advancedArmor core:advancedShield",
                 "repairer": true,
                 "density": 1,
 
-                "guard": {
+                "guard":
+                {
```

Comment:
Any reason behind this change? Every json file has the format
```
asd: {
    ....
}
```
Unless you have good reason, I would revert back.